treeherder/log_parser/crossreference.py

```diff
@@ -72,9 +67,10 @@ def _crossreference(job):
         # We can have a line without a pattern at the end if the log is truncated
         if failure_line is None:
             break
-        logger.error("Failed to match structured line '%s' to an unstructured line" % repr_str)
+        logger.warning("Crossreference %s: Failed to match structured line '%s' to an unstructured line" %
```

Comment:
Well it's helpful if it's failing to have the failures logged in papertrail in case it fails in cases it should handle. UP to you if you don't think it's worth the logspam though.